libkbfs/constants.go

```diff
@@ -15,3 +15,8 @@ const rekeyRecheckInterval = 30 * time.Second
 
 // rekeyInitialTTL is the maximum number rechecks each rekey request can trigger.
 const rekeyInitialTTL = 4
+
+// mdserverReconnectBackoffWindow is a backoff window within which we try to
+// wait randomly for before reconnecting to MD server.
+// TODO: increase this to 5 min
```

Comment:
ah this was accidentally committed as I was testing it. Meant to leave it as 30sec for now